rts/Game/UI/KeyBindings.cpp

```diff
@@ -305,19 +305,15 @@ void CKeyBindings::Kill()
 
 
 /******************************************************************************/
-const CKeyBindings::ActionList& CKeyBindings::GetActionList(const std::string& matcher) const
+const CKeyBindings::ActionList& CKeyBindings::GetActionList() const
 {
 	static ActionList merged; //FIXME switch to thread_local (?)
 	const ActionList* alPtr;
 
 	for (const auto& p: bindings) {
```

Comment:
There could probably be a map of a map to be more efficient, I'm unfamiliar with boost so this is a simple implementation.